client/src/views/TrackViewer/Viewer.vue

```diff
@@ -147,7 +147,10 @@ export default defineComponent({
       }
       // if removed track was selected, unselect before remove
       if (selectedTrackId.value === trackId) {
-        selectNextTrack(1);
+        const newTrack = selectNextTrack(1) !== null ? selectNextTrack(1) : selectNextTrack(-1);
```

Comment:
```suggestion
        const newTrack = selectNextTrack(1) || selectNextTrack(-1);
```
I think this is a little more succinct.  I know I write a lot of ternary myself, but I don't like reading it :P

Reply:
Ah, this won't do.  `0` will evaluate false.  

Better to go with the strict check.

Reply:
Yeah my thought process was exactly the same, started with the || and realized the 0 `trackId ` values.